### PR TITLE
Add eResource access message for library_id logged in users, fixes #214

### LINKED ISSUE #214: patron info: eResource access
Split out from #53 
- blocked until we have LDAP permissions
- This only displays for fee borrowers

### PR DESCRIPTION
**Pull request by mejackreed**
This could have been implemented using the existing `shibboleth?` method, but would not work in local development. I've implemented the `library_id_user` to reduce developer confusion.

Screen Shot 2019-07-25 at 2 30 57 PM

We may also want to adjust markup here (column lengths) but thought that might want to be deferred once we get all of the information needed here.

### CODE REVIEW

File: app/views/patron/_patron.html.erb [hunk 1]
```diff
@@ -10,6 +10,10 @@
     <dt class="col-2 col-sm-1 font-weight-normal">Borrower limit:</dt>
     <dd class="col-10 col-sm-11 patron-status"><%= pluralize(patron.borrow_limit, 'checkout') %></dd>
   <% end %>
+  <% if current_user.library_id_user? %>
```
Comment (cbeer) [inline app/views/patron/_patron.html.erb]: Per @saseestone: we should tie this to `patron.fee_borrower?`, not how they logged in.

Comment (mejackreed) [reply app/views/patron/_patron.html.erb]: That simplifies things :) ! 

I've pushed up the change and removed the extra logic I added.

File: app/views/patron/_patron.html.erb [hunk 2]
```diff
@@ -10,6 +10,10 @@
     <dt class="col-2 col-sm-1 font-weight-normal">Borrower limit:</dt>
     <dd class="col-10 col-sm-11 patron-status"><%= pluralize(patron.borrow_limit, 'checkout') %></dd>
   <% end %>
+  <% if patron.fee_borrower? %>
+    <dt class="col-2 col-sm-1 font-weight-normal">eResource access:</dt>
```
Comment (camillevilla) [inline app/views/patron/_patron.html.erb]:
```suggestion
    <dt class="col-4 col-md-2 font-weight-normal">eResource access:</dt>
```

File: app/views/patron/_patron.html.erb [hunk 3]
```diff
@@ -10,6 +10,10 @@
...
+    <dd class="col-10 col-sm-11 patron-status">In-library only</dd>
```
Comment (camillevilla) [inline app/views/patron/_patron.html.erb]:
```suggestion
    <dd class="col-8 col-md-10 patron-status">In-library only</dd>
```

**Pull request merged**